sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -278,7 +280,7 @@ def get_copy_authorization(self, resource_id, resource_region, **kwargs):
         :param str resource_id: Azure Resource Id of the target Form Recognizer resource
             where the model will be copied to.
         :param str resource_region: Location of the target Form Recognizer resource. A valid Azure
-            region name supported by Cognitive Services.
+            region name supported by Cognitive Services. For example, 'westus', 'centraluseuap' etc.
```

Comment:
I don't want to suggest people create FR resources in canary. Let's change 'centraluseuap' to like 'eastus' or something

Reply:
Let's also add this link that shows regional availability: https://azure.microsoft.com/global-infrastructure/services/?products=cognitive-services